regression/cbmc/r_w_ok8/main.c

```diff
@@ -0,0 +1,9 @@
+#include <assert.h>
+#include <stdlib.h>
+
+int main()
+{
+  int *x = malloc(2);
+  int *y = malloc(2);
+  assert(__CPROVER_r_ok(x, 3) == __CPROVER_r_ok(y, 3));
```

Comment:
:question: Not sure I understand the purpose of the test

Reply:
This was reported here https://github.com/diffblue/cbmc/issues/5194

Reply:
:+1: Please add to the desc file :slightly_smiling_face: